frontend/packages/kubevirt-plugin/integration-tests/v2v/vmware/v2v.vmware.import.incorrect.names.negative.scenario.ts

```diff
@@ -0,0 +1,76 @@
+import { VmwareImportWizard } from '../../tests/models/vmwareImportWizard';
+import { VirtualMachineModel } from '../../../src/models';
+import { vmwareVMConfig } from './v2v.configs';
+import { asyncForEach, click } from '@console/shared/src/test-utils/utils';
+import { wrongValues } from '../../tests/utils/constants/vm';
+import { NetworkInterfaceDialog } from '../../tests/dialogs/networkInterfaceDialog';
+import { clickKebabAction } from '../../views/wizard.view';
+import { KEBAP_ACTION } from '../../tests/utils/constants/common';
+import { browser } from 'protractor';
+import * as view from '../../views/importWizard.view';
+import { DiskDialog } from '../../tests/dialogs/diskDialog';
+
+describe('VMWare Wizard validation, negative tests', () => {
+  const wizard = new VmwareImportWizard();
+
+  beforeEach(async () => {
+    await wizard.openWizard(VirtualMachineModel);
+  });
+
+  afterEach(async () => {
+    await wizard.closeWizard();
+  });
+
+  it('VMWare - Import Wizard shows warning when using incorrect VM name', async () => {
+    await wizard.importVmConnectProviderStep(vmwareVMConfig);
+    await asyncForEach(wrongValues, async (curValue) => {
+      const err = await wizard.fillName(curValue);
+      expect(err).toBeDefined();
+    });
+  });
+
+  it('VMWare - Import Wizard shows warning when using incorrect VM NIC name', async () => {
+    await wizard.importVmConnectProviderStep(vmwareVMConfig);
+    await wizard.importVmConfigStep(vmwareVMConfig);
+    // Starting negative test for VM's NIC name
+    const importedNICs = await wizard.getImportedNics();
+    const nicDialog = new NetworkInterfaceDialog();
+    if (importedNICs.length > 0) {
+      const nic = importedNICs[0];
+      await clickKebabAction(nic.name, KEBAP_ACTION.Edit);
+      await asyncForEach(wrongValues, async (curValue) => {
+        const err = await nicDialog.fillName(curValue);
+        expect(err).toBeDefined();
+      });
+      await browser.sleep(10000);
```

Comment:
are all of these sleeps necessary?

Reply:
Did that for debug purpose, will remove them